setpointRate: 200
motorSimRate: 200
pidControlRate: 200
motorJointRate: 200

motorSimSampleTime: 0.01
motorJointSampleTime: 0.01
control_sample_time: 0.01

control_kp: 0.095
control_ki: 0.05

setPoint_Amplitude: 5.0
setPoint_Freq: 0.1